packages/web3-eth-accounts/src/account.ts

```diff
@@ -171,10 +267,23 @@ export const recoverTransaction = (rawTransaction: HexString): Address => {
 
 /**
  * Recovers the Ethereum address which was used to sign the given data
- *
- * @param data
- * @param signature
- * @param hashed
+ * @param data - Either a signed message, hash, or the {@link signatureObject}
+ * @param signature - The raw RLP encoded signature
+ * @param hashed - If the message is hashed or not
```

Comment:
Last param of recover should be `preFixed` Boolean according to 1.x doc and code https://github.com/ChainSafe/web3.js/blob/aae9d4a9a4ab6d68296f0cc1a2a29540eb3ff433/packages/web3-eth-accounts/src/index.js#L487   :

`preFixed - Boolean (optional, default: false): If the last parameter is true, the given message will NOT automatically be prefixed with "\x19Ethereum Signed Message:\n" + message.length + message, and assumed to be already prefixed.`